connectors/tests/ftest.sh

```diff
@@ -15,11 +15,13 @@ $ROOT_DIR/bin/fake-kibana --index-name search-$NAME --service-type $NAME --debug
 
 make load-data
 
-$ROOT_DIR/bin/elastic-ingest --one-sync --sync-now --debug
+$ROOT_DIR/bin/python -m memray run -f -o $NAME-sync.bin $ROOT_DIR/bin/elastic-ingest --one-sync --sync-now --debug
```

Comment:
can we make it optional? I am thinking 

```
make ftest NAME=mysql MEMPROFILE=yes
```

and by default we don't do it
so we can also use the script in https://github.com/elastic/connectors-python/pull/180 
and allow:

```
make ftest NAME=mysql PERF_METRICS=yes
```